strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/config/PyPiArtifactGeneratorTestConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.carlspring.strongbox.config;
+
+import org.carlspring.strongbox.cron.config.CronTasksConfig;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import({
+        CommonConfig.class,
+        TestingCoreConfig.class,
+        EventsConfig.class,
+        CronTasksConfig.class,
+        StorageCoreConfig.class,
+        StorageApiConfig.class,
+        ClientConfig.class,
+        PypiLayoutProviderConfig.class,
+})
+public class PyPiArtifactGeneratorTestConfig {
```

Comment:
Cosmetic: Braces on new lines, please.

Reply:
Also, why do you actually need this class? I think you should drop it. You shouldn't need tese contexts for the artifact generator.